llvm/docs/LangRef.rst

```diff
@@ -3413,33 +3412,38 @@ For a simpler introduction to the ordering constraints, see the
     stronger) operations on the same address. If an address is written
     ``monotonic``-ally by one thread, and other threads ``monotonic``-ally
     read that address repeatedly, the other threads must eventually see
-    the write. This corresponds to the C++0x/C1x
-    ``memory_order_relaxed``.
+    the write. This corresponds to the C++/C ``memory_order_relaxed``.
```

Comment:
Nit: C/C++ is the usual formulation you see here, not C++/C

Reply:
Done.

Reply:
I was puzzled by `C++/C` formulation as well but was thinking whether jyknight wants to emphasize that C atomics  were based on C++'s :)